packages/orbit-components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -78,108 +69,77 @@ export const WithTextLinkInLabel = ({ checked }) => {
           .
         </Text>
       }
-      checked={checked}
-      value="value"
-      onChange={action("changed")}
     />
-  );
-};
-
-WithTextLinkInLabel.story = {
-  name: "With TextLink in label",
+  ),
 
   parameters: {
     info: "Additionally you can add info to this component.",
+    controls: {
+      exclude: ["label"],
+    },
   },
-};
 
-WithTextLinkInLabel.args = {
-  checked: true,
+  args: {
+    checked: true,
+  },
 };
 
-export const WithTooltip = () => {
-  return (
-    <Checkbox
-      label="Direct"
-      value="one"
-      disabled
-      tooltip={
-        <Tooltip content="There are no results available with this option" placement="top" />
-      }
-    />
-  );
-};
+export const WithTooltip: Story = {
+  render: args => {
+    return (
+      <Checkbox
+        {...args}
+        tooltip={
+          <Tooltip content="There are no results available with this option" placement="top" />
+        }
+      />
+    );
+  },
+
+  args: {
+    disabled: true,
+  },
 
-WithTooltip.story = {
   parameters: {
-    info: "Additionally you can add tooltip to this component.",
+    control: {
+      exclude: ["onChange"],
+    },
   },
 };
 
-export const Playground = ({
-  label,
-  value,
-  checked,
-  disabled,
-  hasError,
-  readOnly,
-  info,
-  dataTest,
-  name,
-}) => {
-  return (
-    <Checkbox
-      label={label}
-      value={value}
-      checked={checked}
-      disabled={disabled}
-      name={name}
-      hasError={hasError}
-      dataTest={dataTest}
-      info={info}
-      readOnly={readOnly}
-      onChange={action("changed")}
-    />
-  );
-};
+export const ReadOnly: Story = {
+  args: {
+    readOnly: true,
+  },
 
-Playground.story = {
   parameters: {
-    info: "Playground of Checkbox",
+    control: {
+      exclude: ["onChange"],
```

Comment:
Hm, it's strange that you excluded this but it's still in the controls there 😕

Reply:
Typo, it should be `controls` instead of `control`. 🙈